gnc/pmc/include/pmc/fam.h

```diff
@@ -21,6 +21,8 @@
 
 #include <Eigen/Dense>
 
+#include <ff_common/ff_ros.h>
```

Comment:
we shouldn't need this here. put in C++ file to reduce compile times

Reply:
now we need it because of the logger macro :). But the config_reader was a dependency already, which depends on ff_common, so it shouldn't be too bad